config/initializers/rollbar.rb

```diff
@@ -0,0 +1,85 @@
+require 'rollbar/rails'
+
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test' and dev:
+  if !Rails.env.production?
```

Comment:
thoughts on enabling it based on an env var instead? I am imagining a scenario in which someone forks this but is not using rollbar in prod.

Reply:
Yes, I've just smoke tested it, and the rollbar agent doesn't complain in the regular log if it has no config var. When an exception occurs, it does add to the logs:
```
[Rollbar] Got unexpected status code from Rollbar api: 400
[Rollbar] Response: {
  "err": 1,
  "message": "access token required"
}
```